Mage.Sets/src/mage/cards/h/HistoriansWisdom.java

```diff
@@ -0,0 +1,93 @@
+import mage.abilities.condition.Condition;
+import mage.abilities.condition.common.AttachedToMatchesFilterCondition;
+import mage.abilities.decorator.ConditionalContinuousEffect;
+import mage.abilities.decorator.ConditionalInterveningIfTriggeredAbility;
+import mage.abilities.effects.common.DrawCardSourceControllerEffect;
+import mage.abilities.effects.common.continuous.BoostEnchantedEffect;
+import mage.constants.ComparisonType;
+import mage.constants.SubType;
+import mage.abilities.effects.common.AttachEffect;
+import mage.constants.Outcome;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.mageobject.PowerPredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.target.TargetPermanent;
+import mage.abilities.keyword.EnchantAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+
+/**
+ *
+ * @author weirddan455
+ */
+public final class HistoriansWisdom extends CardImpl {
+
+    private static final Condition creatureCondition = new AttachedToMatchesFilterCondition(StaticFilters.FILTER_PERMANENT_CREATURE);
+
+    public HistoriansWisdom(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ENCHANTMENT}, "{2}{G}");
+
+        this.subtype.add(SubType.AURA);
+
+        // Enchant artifact or creature
+        TargetPermanent auraTarget = new TargetPermanent(StaticFilters.FILTER_PERMANENT_ARTIFACT_OR_CREATURE);
+        this.getSpellAbility().addTarget(auraTarget);
+        this.getSpellAbility().addEffect(new AttachEffect(Outcome.BoostCreature));
+        this.addAbility(new EnchantAbility(auraTarget.getTargetName()));
+
+        // When Historian's Wisdom enters the battlefield, if enchanted permanent is a creature with the greatest power among creatures on the battlefield, draw a card.
+        this.addAbility(new ConditionalInterveningIfTriggeredAbility(
+                new EntersBattlefieldTriggeredAbility(new DrawCardSourceControllerEffect(1)),
+                HistoriansWisdomCondition.instance,
+                "When {this} enters the battlefield, if enchanted permanent is a creature with the greatest power among creatures on the battlefield, draw a card."
+        ));
+
+        // As long as enchanted permanent is a creature, it gets +2/+1.
+        this.addAbility(new SimpleStaticAbility(new ConditionalContinuousEffect(
+                new BoostEnchantedEffect(2, 1),
+                creatureCondition,
+                "As long as enchanted permanent is a creature, it gets +2/+1"
+        )));
+    }
+
+    private HistoriansWisdom(final HistoriansWisdom card) {
+        super(card);
+    }
+
+    @Override
+    public HistoriansWisdom copy() {
+        return new HistoriansWisdom(this);
+    }
+}
+
+enum HistoriansWisdomCondition implements Condition {
+    instance;
+
+    @Override
+    public boolean apply(Game game, Ability source) {
```

Comment:
try adding `game.applyEffects()` here

Reply:
That fixed it.  I feel like I've gotten fussed at for calling that method before but I can't remember the context.  Maybe it's fine here.

Reply:
yeah there are definitely places where you shouldn’t call it but it’s basically required here, although it’s possible that it should be called somewhere deeper in the code